.circleci/images/Dockerfile

```diff
@@ -1,13 +1,21 @@
-FROM cimg/python:3.8
+FROM cimg/base:stable-20.04
 ARG DEBIAN_FRONTEND=noninteractive
+USER root
+RUN apt-get update
+
+# Utilities already installed
+# Install Python
+
+RUN apt-get install -y python3.8
```

Comment:
Current version of Python is 3.9. Any particular reason for picking 3.8? If not, let's move to 3.9.

Reply:
Actually 3.8 is fine - cancel that.